scripts/sync-council-table.js

```diff
@@ -0,0 +1,112 @@
+require("dotenv").config();
+const mongodb = require("mongodb");
+const { logEmitter, INFO } = require("../src/services/logging.service");
+const {connectToDb, closeConnection} = require("../src/db/db");
+const {getCouncilByUrl, createCouncil } = require("../src/connectors/registrationDb/registrationDb");
+
+let client;
+let configDB;
+let lcConfigCollection;
+
+const establishConnectionToMongo = async () => {
+  client = await mongodb.MongoClient.connect(process.env.CONFIGDB_URL, {
+    useNewUrlParser: true
+  });
+
+  configDB = client.db("register_a_food_business_config");
+
+  lcConfigCollection = configDB.collection("lcConfig");
+};
+
+const getLocalCouncils = async () => {
+  let localCouncils = null;
+  logEmitter.emit("functionCall", "sync-council-table", "getLocalCouncils");
+
+  try {
+    await establishConnectionToMongo();
+
+    localCouncils = await lcConfigCollection
+      .find({
+        $and: [
+          { local_council_url: { $ne: "" } },
```

Comment:
Am I right in thinking this section only grabs the local councils (those with a url field) ? Do we not want to move the county councils across as well?

Reply:
Correct, removed this filter.  Thanks